tests/shutdown/shutdown.pm

```diff
@@ -25,7 +25,12 @@ use power_action_utils 'power_action';
 use utils;
 
 sub run {
-    power_action('poweroff');
+    if (check_var('ARCH', 's390x')) {
```

Comment:
there was any reason to not use [this needle](https://openqa.suse.de/tests/4811285#step/shutdown/1) to detect it? because we mark it as soft-failure when [there isn't](https://openqa.suse.de/tests/4811285#step/shutdown/1)

Reply:
As long as bug is opened, we are fine to record soft failure. If it would pollute results too much, we can add smarter detection, or as proposed in the ticket, do same steps as we do in `first_boot` module.

Reply:
> Don't you mind to add same changes, like in your previous PR to update bug reference in the polkit workaround?

The update of bug reference was included in that [revert](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11178/files)

Reply:
Yes, and as you have added both changes as a single commit, we have reverted both changes and as it's good change, let's apply it here.

Reply:
I mean, I don't know how it happened, but the right code is already in place https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/d1a07057f2e480396b32d6dfaffba908e152195d/lib/opensusebasetest.pm#L1003

Reply:
It's old code, you have renamed it using `bsc1177446` and not `bsc1157928`

Reply:
Oh...

Reply:
> there was any reason to not use [this needle](https://openqa.suse.de/tests/4804965#step/shutdown/5) to detect it? because we mark it as soft-failure when [there isn't](https://openqa.suse.de/tests/4811285#step/shutdown/1)

We do not use this needle because we exit before. As it seems to be a glitch, it cannot be consistently caught.